src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AuthenticationAgentContinuationHelper.cs

```diff
@@ -78,6 +78,11 @@ public static void SetAuthenticationAgentContinuationEventArgs(int requestCode,
             {
                 WebviewBase.SetAuthorizationResult(authorizationResult);
             }
+            else
+            {
```

Comment:
assuming this will only be a case when the broker response are ResponseReceived, BrowserCodeError or UserCancelled, perhaps the below result should be set that way? and the error message should indicate that?

Reply:
i'll need to think about this more. when trying stuff out, i was getting a null authResult and then things were blowing up. but let me think on this more.

---
In reply to: [286275427](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1598#discussion_r286275427) [](ancestors = 286275427)